api/lib/domain/read-models/CleaCertifiedCandidate.js

```diff
@@ -43,10 +43,21 @@ class CleaCertifiedCandidate {
   get isBornInFrenchOutermostRegion() {
     const INSEECodeFrenchOutermostRegion = ['97', '98'];
 
-    return (
-      this.birthINSEECode.startsWith(INSEECodeFrenchOutermostRegion[0]) ||
-      this.birthINSEECode.startsWith(INSEECodeFrenchOutermostRegion[1])
-    );
+    if (this.birthPostalCode) {
+      return (
+        this.birthPostalCode.startsWith(INSEECodeFrenchOutermostRegion[0]) ||
+        this.birthPostalCode.startsWith(INSEECodeFrenchOutermostRegion[1])
+      );
+    }
+
+    if (this.birthINSEECode) {
+      return (
+        this.birthINSEECode.startsWith(INSEECodeFrenchOutermostRegion[0]) ||
+        this.birthINSEECode.startsWith(INSEECodeFrenchOutermostRegion[1])
+      );
+    }
+
+    return false;
```

Comment:
A voir si c'est plus lisible mais c'est plus court
```suggestion
   const isCodeStartsWithFrenchOutermostRegion = 
     (birthCode) => INSEECodeFrenchOutermostRegion.some((frenchOutermostRegion) => birthCode.startsWith(frenchOutermostRegion));
   return [this.birthPostalCode, this. birthINSEECode].some((code) => isCodeStartsWithFrenchOutermostRegion);
```

Reply:
Cool

Reply:
Ah oui mieux !